js/renovation/ui/scroll_view/scrollable_simulated.tsx

```diff
@@ -1171,10 +1175,10 @@ export class ScrollableSimulated extends JSXComponent<ScrollableSimulatedPropsTy
     }
 
     // T320141
-    const isOverflowHidden = getElementComputedStyle(this.contentRef.current)!.overflowX === 'hidden';
+    const computedStyle = getElementComputedStyle(this.contentRef.current);
```

Comment:
The 'getElementComputedStyle' result is not null/undefined if its argument is not null/undefined.
The 'this.contentRef.current' variable is passed as its argument and there is this code few lines above:
```
    if (!isDefined(this.contentRef?.current)) {
      return 0;
    }
```
This member will not null/undefined and the getElementComputedStyle function result will not be null/undefined.
Why you want to change this line of code?

Reply:
Let's extract this change to a new PR with change to window.getComputedStyle direct call.